rust/alluxiocommon/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "alluxiocommon"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+name = "alluxiocommon"
+# "cdylib" is necessary to produce a shared library for Python to import from.
+#
+# Downstream Rust code (including code in `bin/`, `examples/`, and `tests/`) will not be able
+# to `use string_sum;` unless the "rlib" or "lib" crate type is also included, e.g.:
+# crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib"]
+
+[dependencies]
+reqwest = { version = "0.11", features = ["blocking"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
full tokio is a big stuff, probably we only need a specific part of it?

Reply:
understood, will check the related pieces to check in